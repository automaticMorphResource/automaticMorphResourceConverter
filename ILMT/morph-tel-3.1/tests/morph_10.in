1	AhAraM	unk	
1	pUrwigA	unk	
1	jIrNaM	unk
1	ayyi	unk	
1	malabaxXaka	unk	
1	samasya	unk	
1	wIruwuMxi	unk	
1	.	unk	
