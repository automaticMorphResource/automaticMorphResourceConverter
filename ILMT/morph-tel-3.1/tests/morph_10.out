1	AhAraM	unk	<fs af='AhAraM,n,,sg,,d,0,0'>
2	pUrwigA	unk	<fs af='pUrwigA,avy,,,,,0,0_avy'>|<fs af='pUrwi,n,,sg,,,gA,gA'>
3	jIrNaM	unk	<fs af='jIrNaM,n,,sg,,d,0,0'>
4	ayyi	unk	<fs af='avvu,v,any,any,any,,i,i'>
5	malabaxXaka	unk	<fs af='malabaxXakaM,n,,sg,,o,ti,ti'>
6	samasya	unk	<fs af='samasya,n,,sg,,d,0,0'>|<fs af='samasya,n,,sg,,o,ti,ti'>
7	wIruwuMxi	unk	<fs af='wIru,v,fn,sg,3,,wA,wA'>|<fs af='wIru,v,fn,sg,3,,wA,wA'>
8	.	unk	<fs af='&dot;,punc,,,,,,'>
